Tests/AdyenTests/Card Tests/3DS2 Component/ThreeDS2ClassicActionHandlerTests.swift

```diff
@@ -269,4 +274,23 @@ class ThreeDS2ClassicActionHandlerTests: XCTestCase {
         waitForExpectations(timeout: 2, handler: nil)
     }
 
+    // MARK: - Private
+
+    func compareJSONStrings(_ jsonString1: String, _ jsonString2: String) -> Bool {
+        guard let data1 = jsonString1.data(using: .utf8),
+              let data2 = jsonString2.data(using: .utf8) else {
+            print("Invalid data.")
+            return false
+        }
+
+        do {
+            let dict1 = try JSONSerialization.jsonObject(with: data1, options: [.allowFragments]) as? [String: Any]
```

Comment:
try to unwrap it to catch the case where both json strings cannot be casted to [String: Any]
```suggestion
            let dict1 = try XCTUnwrap(JSONSerialization.jsonObject(with: data1, options: [.allowFragments]) as? [String: Any])
```

Reply:
Removed this method. By using sortedKeys in the encoders, its not needed anymore.